README.rst

```diff
@@ -11,6 +11,35 @@ https://travis-ci.org/MacPython/scipy-wheels
 The driving github repository is
 https://github.com/MacPython/scipy-wheels
 
+Using the repository
+====================
+
+There are two important branches:
+
+* ``master`` - for building releases;
+* ``daily`` - for daily builds.
+
+Travis-CI builds the ``daily`` branch - er - daily, via a `Travis-CI cron job
+<https://docs.travis-ci.com/user/cron-jobs/>`_ to check that we can build
+against current Scipy master.   When trying to fix builds against master, or
+developing new CI build machinery, *please use the ``develop`` branch*.
```

Comment:
There's no branch named `develop`?

Reply:
Aha - yes - thanks.  I'll also add the instructions about Appveyor.  I forgot we had enabled that.

Reply:
https://github.com/MacPython/scipy-wheels/pull/8